sdk/src/Services/DynamoDBv2/Custom/DocumentModel/TableConfig.cs

```diff
@@ -30,6 +28,13 @@ public class TableConfig
         /// </summary>
         public string TableName { get; set; }
 
+        /// <summary>
```

Comment:
Same comment as above. Explain the default and why a user might want to change the default.

Reply:
Added a `<remarks>` block in https://github.com/aws/aws-sdk-net/compare/1161dc30f02a5a18108c28b0191f21da5080a368..0c2afdec6c32efc6ab9ebd3841a433bf19683181